src/server/scripts/Northrend/Ulduar/Ulduar/boss_assembly_of_iron.cpp

```diff
@@ -15,751 +15,878 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-/* ScriptData
-SDName: Assembly of Iron encounter
-SD%Complete: 60%
-SDComment: chain lightning won't cast, supercharge don't work (auras don't stack from different casters)
-SDCategory: Ulduar - Ulduar
-EndScriptData */
-
 #include "ScriptMgr.h"
+#include "AreaBoundary.h"
 #include "InstanceScript.h"
 #include "MotionMaster.h"
 #include "ObjectAccessor.h"
 #include "ScriptedCreature.h"
 #include "SpellAuras.h"
 #include "SpellMgr.h"
 #include "SpellScript.h"
+#include "Spell.h"
 #include "TemporarySummon.h"
 #include "ulduar.h"
 
-enum AssemblySpells
+enum Spells
 {
     // General
-    SPELL_SUPERCHARGE                            = 61920,
-    SPELL_BERSERK                                = 47008, // Hard enrage, don't know the correct ID.
-    SPELL_KILL_CREDIT                            = 65195, // spell_dbc
+    SPELL_SUPERCHARGE                           = 61920,
+    SPELL_BERSERK                               = 62535,
+    SPELL_KILL_CREDIT                           = 65195, // Serverside spell
 
     // Steelbreaker
-    SPELL_HIGH_VOLTAGE                           = 61890,
-    SPELL_FUSION_PUNCH                           = 61903,
-    SPELL_STATIC_DISRUPTION                      = 44008,
-    SPELL_OVERWHELMING_POWER                     = 64637,
-    SPELL_ELECTRICAL_CHARGE                      = 61902,
+    SPELL_HIGH_VOLTAGE                          = 61890,
+    SPELL_STATIC_DISRUPTION                     = 44008,
```

Comment:
seems should be 61911 instead of 44008

Reply:
also it need be changed in spelldifficulty_dbc ;)
```sql
DELETE FROM `spelldifficulty_dbc` WHERE `id` IN (44008,61911);
INSERT INTO `spelldifficulty_dbc`(`id`, `spellid0`, `spellid1`) VALUES
(61911, 61911, 63512);
```

Reply:
This one https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?spell=63494#triggered-by

Did you tested it?

Reply:
Ah yep, 63495, edited ;)

Reply:
Will instance->DoRemoveAurasDueToSpellOnPlayers(sSpellMgr->GetSpellIdForDifficulty(SPELL_STATIC_DISRUPTION, creature));
still work correctly with new spell?

Reply:
No, that one needs to be changed to the triggered aura effect